package.json

```diff
@@ -1,26 +1,31 @@
 {
   "name": "mockcloud",
-  "description": "Mock Cloud",
-  "version": "0.2.71",
+  "description": "an image to mock out N TritonDC compute nodes",
+  "version": "2.0.0",
   "author": "Joyent (joyent.com)",
   "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/joyent/sdc-mockcloud.git"
+  },
   "dependencies": {
     "assert-plus": "^1.0.0",
-    "bunyan": "1.4.0",
-    "cmdln": "^3.5.3",
+    "bunyan": "^1.8.12",
     "cn-agent": "git+https://github.com/joyent/sdc-cn-agent.git#master",
-    "mockery": "1.4.0",
-    "node-uuid": "1.4.3",
-    "restify": "4.0.0",
-    "tabula": "^1.7.0",
-    "tape": "^4.4.0",
-    "ur-agent": "git+https://github.com/joyent/sdc-ur-agent.git#555bbf161fa908272b23ca79285d39b89abaefa1",
-    "sdc-clients": "git+https://github.com/joyent/node-sdc-clients.git#8e52debc25b3239b62772076cb2c4ef70fb2113d",
-    "vasync": "^1.6.3",
-    "verror": "^1.6.1"
+    "sdc-clients": "^11.3.0",
+    "uuid": "^3.2.1",
+    "vasync": "^2.2.0",
+    "verror": "^1.10.0"
+  },
+  "devDependencies": {
+    "eslint": "4.13.1",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-joyent": "~2.0.0",
+    "eslint-plugin-prettier": "^2.6.0",
+    "prettier": "^1.11.1"
   },
   "engines": {
-    "node": ">=0.10"
+    "node": ">=4.x"
```

Comment:
I'm inclined to leave it at 4.x for now: v6 was used in the Makefile because that's what cn-agent uses currently. However, vm-agent uses 4.x for now. config-agent uses 0.10, but I think that can be updated to v4 or v6 soonish.